src/main/kotlin/com/cognifide/gradle/aem/instance/tasks/InstanceRestoreAndUp.kt

```diff
@@ -0,0 +1,14 @@
+package com.cognifide.gradle.aem.instance.tasks
+
+import com.cognifide.gradle.aem.common.AemDefaultTask
+
+open class InstanceRestoreAndUp : AemDefaultTask() {
```

Comment:
then restore will also incorporate up;  please avoid `andXX` in task names, this is scary and confusing :)

Reply:
OK